### PR TITLE
Added initial committees data for members committee routes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/parliament/data/api/v1/house_committees/index [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/house_committees/index]: Why unicode "\u00D6", should this be "X"

File: lib/parliament/data/api/v1/house_committees_a_to_z/index [hunk 2]
```diff
@@ -0,0 +1,25 @@
+_:node1303551 <http://id.ukpds.org/schema/value> "K" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/house_committees_a_to_z/index:1]: Missing "X"

File: lib/parliament/data/api/v1/house_committees_by_initial/index [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/house_committees_by_initial/index]: Why unicode "\u00D6", should this be "X"

File: lib/parliament/data/api/v1/house_committees_current/index [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/house_committees_current/index]: same as above

File: lib/parliament/data/api/v1/house_committees_current_a_to_z/index [hunk 5]
```diff
@@ -0,0 +1,25 @@
+_:node1303551 <http://id.ukpds.org/schema/value> "K" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/house_committees_current_a_to_z/index:1]: Missing "X"

File: lib/parliament/data/api/v1/house_committees_current_by_initial/index [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/house_committees_current_by_initial/index]: Why unicode "\u00D6", should this be "X"

File: lib/parliament/data/api/v1/person_committees_chairs/index [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/person_committees_chairs/index]: same as above

File: lib/parliament/data/api/v1/person_committees_memberships/index [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/person_committees_memberships/index]: same as above

File: lib/parliament/data/api/v1/person_committees_memberships_current/index [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+_:node1349973 <http://id.ukpds.org/schema/value> "P" .
+_:node1349974 <http://id.ukpds.org/schema/value> "O" .
+_:node1349975 <http://id.ukpds.org/schema/value> "L" .
+_:node1349976 <http://id.ukpds.org/schema/value> "V" .
+_:node1349977 <http://id.ukpds.org/schema/value> "N" .
+_:node1349978 <http://id.ukpds.org/schema/value> "E" .
+_:node1349979 <http://id.ukpds.org/schema/value> "Y" .
+_:node1349980 <http://id.ukpds.org/schema/value> "I" .
+_:node1349981 <http://id.ukpds.org/schema/value> "U" .
+_:node1349982 <http://id.ukpds.org/schema/value> "\u00D6" .
```
Comment (naseberry) [inline lib/parliament/data/api/v1/person_committees_memberships_current/index]: same as above

**Review by naseberry — APPROVED** [review]

**Review by naseberry — APPROVED** [review]

**Review by jennaramdenee — APPROVED** [review]

**Pull request merged**